src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs

```diff
@@ -31,24 +31,6 @@ public partial class View
     {
         internal string styleName;
 
-        [Flags]
-        internal enum BackgroundExtraDataUpdatedFlag : byte
-        {
-            BackgroundCornerRadius = 1 << 0,
-            ShadowCornerRadius = 1 << 2,
-            ContentsCornerRadius = 1 << 3, /// Subclass cases.
-
-            Background = BackgroundCornerRadius,
-            Shadow = ShadowCornerRadius,
-
-            CornerRadius = BackgroundCornerRadius | ShadowCornerRadius | ContentsCornerRadius,
-
-            None = 0,
-            All = Background | Shadow,
-        }
-
-        internal BackgroundExtraDataUpdatedFlag backgroundExtraDataUpdatedFlag = BackgroundExtraDataUpdatedFlag.None;
```

Comment:
VD might use this value internally. I'll check it whether we should keep it or removable.